backend/src/api/api-routes.openapi.ts

```diff
@@ -217,13 +217,21 @@ const spec: { paths: OpenAPIV3.PathsObject } = {
                                 type: "object",
                                 required: ["type", "text", "color"],
                                 properties: {
+                                    x: { type: "number" },
+                                    y: { type: "number" },
                                     type: { type: "string", enum: ["note"] },
                                     text: { type: "string" },
                                     color: { type: "string" },
                                     container: { type: "string" },
+                                    width: { type: "number" },
+                                    height: { type: "number" },
+                                    replaceXIfExists: { type: "boolean" },
```

Comment:
I think the replace*IfExists params can be omitted for the new attributes (x,y...). They are only useful for the original use case. Please remove them if you don't see a particular need for them.

Reply:
I mean across the pull request, not just in this location ofc.

Reply:
understood the concern, will check and modify if needed (checking the note updation part with x, y, width, height parameters)

Reply:
(replace*IfExists ) this allows x, y, width and height parameters updated through put request, if you think updation of the note that is created with (x, y, width, height) params is not necessary, we can remove. any suggestions on this bro

Reply:
I suggest that the API updates x..height always when these parameters are supplied. Not need to use the extra replace-flags.